core/templates/dev/head/app.js

```diff
@@ -498,99 +498,6 @@ oppia.factory('oppiaDatetimeFormatter', ['$filter', function($filter) {
   };
 }]);
 
```

Comment:
uh oh, what is all this and below. Did you mess up while merging ?

Reply:
yeah, I messed it up

Reply:
I'm trying to fix it.

Reply:
well, a good way to merge properly is
merge remote develop branch in your local develop branch
then switch back to your branch and merge local develop into it

Now, that it's messed up. Go to a comment before merge (maybe use git reset --hard) and then merge from there.

Reply:
@1995YogeshSharma Yes fixed that, please review.